artistools/spectra.py

```diff
@@ -64,8 +64,8 @@ def get_spectrum(modelpath, timestepmin: int, timestepmax=-1, fnufilterfunc=None
         timestepmax = timestepmin
 
     master_branch = False
-    if os.path.isfile(modelpath / 'specpol.out'):
-        specfilename = modelpath / "specpol.out"
+    if os.path.isfile(Path(modelpath) / 'specpol.out'):
```

Comment:
Eventually, I'd like to replace all uses of os.path with their newer pathlib.Path equivalents (because importing modules is slow and the code will be slightly more readable). For example, os.path.isfile(Path(modelpath) / 'specpol.out') can be replaced with Path(modelpath, 'specpol.out').is_file().